script/Deploy.s.sol

```diff
@@ -1,9 +1,23 @@
-// SPDX-License-Identifier: UNLICENSED
-
+// SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.18;
 
-import "forge-std/Script.sol";
+import {Script} from "forge-std/Script.sol";
+import {DeployInput} from "script/DeployInput.sol";
+import {PooltogetherGovernor} from "src/PooltogetherGovernor.sol";
+
+contract Deploy is DeployInput, Script {
+  uint256 deployerPrivateKey;
+
+  function setUp() public {
+    deployerPrivateKey = vm.envUint("DEPLOYER_PRIVATE_KEY");
```

Comment:
we should have a .env.template file with this var represented there

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/6/commits/7f44d8e4219d164eb2ebb130aadd53a3b50a4c81